gridstatus/tests/base_test_iso.py

```diff
@@ -285,7 +285,7 @@ def _check_time_columns(self, df, instant_or_interval="interval"):
         assert isinstance(df, pd.DataFrame)
 
         if instant_or_interval == "interval":
-            time_cols = ["Time", "Interval Start", "Interval End"]
```

Comment:
@kmax12 Removing this results in a large number of test failures (99). I think we should wait to update this test until we update the source code.

Reply:
okay